### ISSUE TITLE
Question: Probably a really dumb question, but I'm completely stumped on step 2 of the installation process.

### ISSUE THREAD

**Issue by NylaTheWolf** [ArchiveBox/ArchiveBox#759]
I'm on Mac OS, and I downloaded Docker Desktop. (I'm following the recommended method, downloading and setting up archivebox through docker-compose) I went through the tutorial and all. I downloaded the zip file for ArchiveBox. But... now what? The instructions say to put the docker-composer.yml file into an empty directory. Do I do that with the code provided? Where do I put that code? I'd assume in Docker but I can't find any way to make a new container (if I'm even supposed to make a container) or edit the tutorial contain or the repo, which is what the dashboard shows. Do I move docker-composer.yml from the ArchiveBox-dev folder into a new, empty folder?

This is probably a really stupid question but I've literally never used Docker in my life before and I'm not familiar with this stuff.

**Comment by pirate** [comment]
Just copy/paste and run these commands verbatim in terminal, no need to download any Zip or move things manually in finder:
```bash
mkdir ~/archivebox && cd ~/archivebox
curl -O 'https://raw.githubusercontent.com/ArchiveBox/ArchiveBox/master/docker-compose.yml'

docker-compose run archivebox init --setup
# type in the admin user and pass you want (you can just press enter to leave the email empty)

docker-compose up
```
Then open http://127.0.0.1:8000 in your browser.

**Comment by NylaTheWolf** [comment]
@pirate sorry for the late response, I've been busy!

So I entered the commands into Terminal. I had to go to bed so I exited the terminal after doing a certain step (like the 2nd). I did docker-compose up
![Screen Shot 2021-06-05 at 7 41 38 PM](https://user-images.githubusercontent.com/41797151/120908395-b8bfa380-c637-11eb-9ffe-ea1d3780f281.png)

Then my dumb ass got worried that I did it wrong or something, so I did `docker-compose run archivebox init --setup`

![Screen Shot 2021-06-05 at 7 49 59 PM](https://user-images.githubusercontent.com/41797151/120908406-dab92600-c637-11eb-8d66-a2e618a4b9f3.png)
And when I tried to create an admin superuser, I must've accidentally run the command again?

Now `archivebox_1  | "GET /admin/login/ HTTP/1.1" 200 11144` keeps popping up.

Did I screw something up?
